lib/runner/request-helpers-postsend.js

```diff
@@ -23,23 +24,17 @@ module.exports = [
 
         // invoke `post` on the Auth
         authHandler.post(authInterface, context.response, function (err, success) {
+            // sync auth state back to item request
+            _.set(context, 'item.request.auth', auth);
+
             // there was an error in auth post hook
             if (err) { return done(err); }
 
             // auth was verified
-            if (success) {
-                return done();
-            }
-
-            // Auth wants to replay the request
-            var replayController = new ReplayController(context.replayState, run);
-
-            // finish up current request and schedule a replay
-            replayController.requestReplay(context, context.item, false, function (err) {
-                done(err);
-            }, function (err) {
-                done(err);
-            });
+            if (success) { return done(); }
```

Comment:
Shouldn't this be an explicit `replay: false`?

Reply:
IMHO, the replays should be an opt-in flow. I am not sure about forcing every other flow to turn off the. The way I see it, I want more options to be introduced. In that case wouldn't it be too much of an overhead?